catalog-info.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: homes-go-common
+  description: Homes shared go library
```

Comment:
I think you can add a little bit more info, like what this library do, what's intention, e.t.c.